### PR TITLE
Add media gallery images to content pages; set page image widths to 720px

### PR DESCRIPTION
**Pull request by B77Mills**

### CODE REVIEW

File: sites/aviationpros.com/server/graphql/fragments/content-list.js [hunk 1]
```diff
@@ -44,6 +44,20 @@ fragment WebsiteContentListFragment on Content {
       }
     }
   }
+  ... on MediaGallery {
+    images {
```
Comment (zarathustra323) [inline sites/aviationpros.com/server/graphql/fragments/content-list.js]: This shouldn’t be on content lists. This is also the wrong type. it should be ContentMediaGallery

**Comment by zarathustra323** [comment]
I’m pretty sure none of those content list fragments should change at all.

**Comment by B77Mills** [comment]
I removed the images from the content-list fragment cause they are not needed.  

As for the errors..... OUR WORK INTERNET SUCKS!!!!!  They are graph timeouts on related and section content

File: sites/vehicleservicepros.com/server/graphql/fragments/content-list.js [hunk 2]
```diff
@@ -29,6 +29,18 @@ fragment WebsiteContentListFragment on Content {
     alt
     isLogo
   }
+  images {
```
Comment (zarathustra323) [inline sites/vehicleservicepros.com/server/graphql/fragments/content-list.js]: Remove this change.

**Pull request merged**